### PR TITLE
Added EC2 vpcId, subnetId parameters. Fixes #91

### LINKED ISSUE #91: Support issue with EC2 provisioner and AWS EC2 Classic
I'm attempting to install inlets-operator by way of arkade which results in:

```
$ kubectl logs -n inlets deploy/inlets-operator
2020/08/08 19:10:49 Inlets client: inlets/inlets:2.7.3
2020/08/08 19:10:49 Inlets pro: false
W0808 19:10:49.685014       1 client_config.go:552] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
I0808 19:10:49.685868       1 controller.go:121] Setting up event handlers
I0808 19:10:49.685900       1 controller.go:243] Starting Tunnel controller
I0808 19:10:49.685903       1 controller.go:246] Waiting for informer caches to sync
I0808 19:10:49.785997       1 controller.go:251] Starting workers
I0808 19:10:49.786007       1 controller.go:257] Started workers
2020/08/08 19:10:49 Creating tunnel for nginx-1-tunnel.default
I0808 19:10:49.789938       1 controller.go:315] Successfully synced 'default/nginx-1'
2020/08/08 19:10:49 Provisioning started with provider:ec2 host:nginx-1-tunnel
E0808 19:10:51.798084       1 controller.go:320] error syncing 'default/nginx-1-tunnel': InvalidParameter: The AssociatePublicIpAddress parameter is only supported for VPC launches.
        status code: 400, request id: 7bc05cdf-7eb5-4167-9f44-3616397a40c6, requeuing
```

Secondary to the above, I had a hard time finding documentation for installing with AWS provider.

## Expected Behaviour

Installing inlets-operator via `arkade` results in no error messages and creates an EC2 instance and whatever other steps should be done as part of what constitutes a successful install.

Documentation should be easier to find and should provide clear steps for a successful install and what to do if there's an issue. The documentation should also explicitly detail what will be done in the providers' account.

## Steps to Reproduce (for bugs)

```
export AWS_PROFILE=default
kubectl create ns inlets
arkade install inlets-operator -n inlets \
    -p ec2 \
    -r us-west-2 \
    -z us-west-2a \
    --token-file ~/Downloads/access-key \
    --secret-key-file ~/Downloads/secret-access-key
```

using arkade version is `0.6.0`

where `access-key` and `secret-access-key` files just contain the access key and secret access key respectively.

## Your Environment

* inlets-operator version, find via `kubectl get deploy inlets-operator -o wide`

* Kubernetes distribution i.e. minikube v0.29.0., KinD v0.5.1, Docker Desktop: `Bare metal 4 node cluster.`

* Kubernetes version `kubectl version`: `1.18.6`

* Operating System and version (e.g. Linux, Windows, MacOS): `Linux (Ubuntu 20.04)`

* Cloud provisioner: `AWS (us-west-2)`

### PR DESCRIPTION
**Pull request by digitalsanctum**
Signed-off-by: Shane Witbeck <shane@digitalsanctum.com>

- [x] I have raised an issue to propose this change.

## Description

This propagates a fix to `inletsctl`. See https://github.com/inlets/inletsctl/pull/75

## How Has This Been Tested?

Not tested yet. Submitting for early feedback.

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/inlets/inlets/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: chart/inlets-operator/values.yaml [hunk 1]
```diff
@@ -26,6 +26,12 @@ accessKeyFile: "/var/secrets/inlets/inlets-access-key"
 # that requires an access key and secret key, such as EC2.
 secretKeyFile: ""
 
+# The VPC ID to create the exit-server in (EC2)
```
Comment (alexellis) [inline chart/inlets-operator/values.yaml:29]: I'm wondering if it'd be clearer / cleaner to namespace this to:

```
ec2:
  vpcId:
  subnetId
```

It depends on whether we'll need this for other providers

**Comment by alexellis** [comment]
Thanks for this PR @digitalsanctum - it looks good to me. I would just ask you to fix the conflict on the go.mod/sum. Yesterday I merged support for a newer inletsctl version, so you should be able to revert any changes to those files, or stash your changes, rebase to master and apply them again.

**Review by alexellis — CHANGES_REQUESTED** [review]
Rebase required please.

File: chart/inlets-operator/values.yaml (hunk 1 above)
Comment (digitalsanctum) [reply chart/inlets-operator/values.yaml:29]: Leaving for now. Happy to address in a subsequent PR if need be.

**Comment by alexellis** [comment]
Any reason for closing this?

**Comment by Waterdrips** [comment]
https://github.com/inlets/inlets-operator/pull/94 for ref this was implemented in a new pr

**Pull request closed without merging**